cartoframes/vis/defaults.py

```diff
@@ -1,30 +1,21 @@
-_HTML_TEMPLATE = '<iframe srcdoc="{srcdoc}" width="{width}" height="{height}"></iframe>'
+CARTO_VL_PATH = 'https://libs.cartocdn.com/carto-vl/v1.2.5/carto-vl.min.js'
```

Comment:
IMO we should turn to just v1.2 for free updates from our cdn

Reply:
:+1:

Reply:
Added here: https://github.com/CartoDB/cartoframes/pull/680/commits/78598d139983375345310c79fcf90a95384270c6